src/main/java/uk/gov/hmcts/reform/pip/subscription/management/errorhandling/GlobalExceptionHandler.java

```diff
@@ -34,4 +37,39 @@
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
 
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<ExceptionResponse> handleArgumentNotValid(
+        MethodArgumentNotValidException ex) {
+        ExceptionResponse exceptionResponse = new ExceptionResponse();
+        StringBuilder responseText = new StringBuilder("Bad Request: ");
+        for (int i = 0; i < ex.getBindingResult().getErrorCount(); i++) {
```

Comment:
I think it would be worth adding a javadoc to this method (and the one below) with some example output formats for this - just to highlight what the string builder here is doing

Reply:
Done!